Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -233,9 +251,11 @@ extension MongoCollection {
 
         let opts = try encoder.encode(options)
         var error = bson_error_t()
+        let reply = Document()
```

Comment:
I don't think that was a problem. On error, you were still reporting the error string from libmongoc's `bson_error_t`. With this change, you're now surfacing the original command error (if any).

The reply was never really needed, since the success case is just returning a list of index names, which can be derived from the input models.